Kickstarter-iOS/Views/Cells/RewardCell.swift

```diff
@@ -11,437 +11,134 @@ protocol RewardCellDelegate: AnyObject {
-  private let descriptionTitleLabel = UILabel(frame: .zero)
-  private let includedItemsStackView = UIStackView(frame: .zero)
-  private let includedItemsTitleLabel = UILabel(frame: .zero)
-  private let minimumPriceConversionLabel = UILabel(frame: .zero)
-  private let minimumPriceLabel = UILabel(frame: .zero)
-  private let pledgeButton: MultiLineButton = {
-    MultiLineButton(type: .custom)
-      |> \.translatesAutoresizingMaskIntoConstraints .~ false
-  }()
-
-  private lazy var pillCollectionView: UICollectionView = {
-    UICollectionView(
-      frame: .zero,
-      collectionViewLayout: PillLayout(
-        minimumInteritemSpacing: Styles.grid(1),
-        minimumLineSpacing: Styles.grid(1),
-        sectionInset: UIEdgeInsets(topBottom: Styles.grid(1))
-      )
-    )
-      |> \.backgroundColor .~ UIColor.white
-      |> \.contentInsetAdjustmentBehavior .~ UIScrollView.ContentInsetAdjustmentBehavior.always
-      |> \.dataSource .~ self.pillDataSource
-      |> \.delegate .~ self
-      |> \.isHidden .~ true
-      |> \.translatesAutoresizingMaskIntoConstraints .~ false
-  }()
-
-  private lazy var pillCollectionViewHeightConstraint: NSLayoutConstraint = {
-    self.pillCollectionView.heightAnchor.constraint(equalToConstant: 0)
-      |> \.priority .~ .defaultHigh
-  }()
-
-  private let pledgeButtonLayoutGuide = UILayoutGuide()
-  private let priceStackView = UIStackView(frame: .zero)
-  private let rewardTitleLabel = UILabel(frame: .zero)
+  internal let rewardCardContainerView = RewardCardContainerView(frame: .zero)
   private let scrollView = UIScrollView(frame: .zero)
-  private let stateImageView: UIImageView = {
-    UIImageView(frame: .zero)
-      |> \.translatesAutoresizingMaskIntoConstraints .~ false
-  }()
+  private lazy var longPressGestureRecognizer: UILongPressGestureRecognizer = {
+    let longPressGestureRecognizer = UILongPressGestureRecognizer(
+      target: self, action: #selector(RewardCell.depress(_:))
+    )
+    longPressGestureRecognizer.minimumPressDuration = CheckoutConstants.RewardCard.Transition
+      .DepressAnimation.longPressMinimumDuration
+    longPressGestureRecognizer.delegate = self
+    longPressGestureRecognizer.cancelsTouchesInView = false
 
-  private let stateImageViewContainer: UIView = {
-    UIView(frame: .zero)
-      |> \.isHidden .~ true
-      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+    return longPressGestureRecognizer
   }()
 
-  private let titleStackView = UIStackView(frame: .zero)
-
-  // MARK: - Lifecycle
-
   override init(frame: CGRect) {
     super.init(frame: frame)
 
     self.configureViews()
-    self.setupConstraints()
-    self.bindViewModel()
 
-    self.pledgeButton.addTarget(self, action: #selector(self.pledgeButtonTapped), for: .touchUpInside)
-    self.containerView.addGestureRecognizer(
-      UITapGestureRecognizer(target: self, action: #selector(self.rewardCardTapped))
-    )
+    self.bindViewModel()
```

Comment:
Looks like we don't have a view model for this cell anymore, so we don't need this.